src/utils/formatting.ts

```diff
@@ -38,3 +38,7 @@ export const constructLink = (link: string): string => {
 
   return link + search;
 };
+
+export const splitIntoRowChunks = (text: string): string[] => {
+  return text.replaceAll(/\\r\\n|\\n|\r\n|<br>|<br\/>/g, '\n').split('\n');
+};
```

Comment:
@IlievaDayana You could add some test for this function as well `splitIntoRowChunks`

Reply:
there is, it is just a feature test that is testing both visually and by calling this function and asserting that the states are equal